runtime/onert/backend/trix/ops/BulkLayer.cc

```diff
@@ -49,24 +50,57 @@ void BulkLayer::configure(const std::vector<const IPortableTensor *> &inputs,
     throw std::runtime_error("Unable to extract the model metadata");
   }
 
+  _model_id.resize(_dev_context->getDevSize());
+
   generic_buffer model_file;
   model_file.type = BUFFER_FILE;
   model_file.filepath = binary_path.c_str();
   model_file.size = _meta->size;
 
-  if (registerNPUmodel(dev_context->getDev(), &model_file, &_model_id) < 0)
+  if (registerNPUmodel(dev_context->getDev(0), &model_file, &_model_id[0]) < 0)
+  {
+    throw std::runtime_error("Failed to register npu model");
+  }
+  if (registerNPUmodel(dev_context->getDev(1), &model_file, &_model_id[1]) < 0)
   {
     throw std::runtime_error("Failed to register npu model");
   }
 }
 
+void single_job(npudev_h dev, int req_id, input_buffers *input_buf, tensors_data_info *in_info,
+                output_buffers *output_buf, tensors_data_info *out_info)
+{
+  if (setNPU_requestData(dev, req_id, input_buf, in_info, output_buf, out_info))
+  {
+    throw std::runtime_error("Unable to create NPU request for red_id (" + std::to_string(req_id) +
+                             ")");
+  }
+
+  if (submitNPU_request(dev, req_id))
+  {
+    throw std::runtime_error("Unable to submit NPU request with req id (" + std::to_string(req_id) +
+                             ")");
+  }
+}
+
 void BulkLayer::run()
 {
-  int req_id;
-  if (createNPU_request(_dev_context->getDev(), _model_id, &req_id))
+  // TODO: Remove too many assumption
+  // We assume user wants batch execurion if user's input size is larger than model's input size
```

Comment:
```suggestion
  // We assume user wants batch execution if user's input size is larger than model's input size
```

Personally, I would like to use stricter condition for assuming batch execution.
- Not just user's input size > model's input size
- But user's input size is multiples of model's input size